app/adapters/explorer-resource.js

```diff
@@ -77,25 +73,17 @@ var ExplorerResourceAdapter = DS.RESTAdapter.extend({
     (Most Riak cluster resources do not have globally unique IDs.
     For example, bucket types are only unique within a cluster.)
     */
-    normalizeId: function(record, type, query, idKey) {
-        var prefix = [];
-        if(query.clusterId) {
-            prefix.push(query.clusterId);
-        }
-        if(query.bucketTypeId && type.modelName !== 'bucket-type') {
-            prefix.push(query.bucketTypeId);
-        }
-        if(!idKey) {
-            idKey = 'id';
-        }
-        var originalId = record[idKey];
-        record.original_id = originalId;
-        prefix.push(originalId);
-        var compositeId = prefix.join('/');
```

Comment:
See my previous comment on the commits. Getting rid of these lines (defining an intermediate `compositeId` variable, etc) is more compact, but is removing the explanation. (The fact that the new record.id is a composite id, etc). We should at least carry over this information in line comments.